en/docs/reference/synapse-properties/transport-parameters/rabbitmq-transport-parameters.md

```diff
@@ -145,13 +139,6 @@ In case of a network failure or broker shutdown, the Micro Integrator will try t
       The retry count specifies the number of times the Micro Integrator will try to recreate a lost connection. The default retry count is <code>3</code>. That is, the Micro Integrator retries only 3 times.
     </td>
   </tr>
-  <tr>
```

Comment:
Let;s remove ### Connection recovery parameters (optional)  section for rabbitmq listener, as that is not applicable to listener.

Reply:
Removed it as well.